src/test/suite/fixtures/recorded/languages/ruby/changeKey2.yml

```diff
@@ -0,0 +1,23 @@
+languageId: ruby
+command:
+  version: 1
+  spokenForm: change key
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: collectionKey, includeSiblings: false}
+initialState:
+  documentContents: "h = {\"1\": \"one\", \"2\":   \"two\"}"
```

Comment:
I'm not sure why this was recorded with a wrapped `"` and the one above it wasn't.

Reply:
No clue. The vagaries of yaml syntax are not for us mere mortals to understand 😅. Cc/ @somanythings

Reply:
Got it, I might just re-record and see if I can reproduce it.